mobius-core/src/main/java/com/spotify/mobius/MessageDispatcher.java

```diff
@@ -40,6 +40,8 @@
   @Nonnull private final WorkRunner runner;
   @Nonnull private final Consumer<M> consumer;
 
+  private volatile boolean disabled;
```

Comment:
```suggestion
  private volatile boolean disabled = false;
```

It's a little more explicit to define the initial value. Also, I wanted to try github's new suggestion feature. :)